packages/date-picker/src/vaadin-date-picker-overlay-content-mixin.js

```diff
@@ -11,6 +11,7 @@ import { addListener, setTouchAction } from '@vaadin/component-base/src/gestures
 import { MediaQueryController } from '@vaadin/component-base/src/media-query-controller.js';
 import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
 import { dateAfterXMonths, dateEquals, extractDateParts, getClosestDate } from './vaadin-date-picker-helper.js';
+import { dateAllowed } from './vaadin-date-picker-helper.js';
```

Comment:
This is a duplicate import from `vaadin-date-picker-helper.js`, there is already one above on the line 13.

Reply:
Updated